admin-src/DiagnoseResponse/index.js

```diff
@@ -96,39 +105,63 @@ export const DiagnoseResponse = ({ query }) => {
         .firestore()
         .doc(`users/${currentDiagnose.userUID}/requests/${currentDiagnose.id}`)
         .update({
-          updatedAt: firebase.firestore.FieldValue.serverTimestamp()
+          updatedAt: firebase.firestore.FieldValue.serverTimestamp(),
+        })
+    }
+
+    const setAdminIsLastAnswer = async () => {
+      await firebase
+        .firestore()
+        .doc(`users/${currentDiagnose.userUID}/requests/${currentDiagnose.id}`)
+        .update({
+          adminIsLastAnswer: true,
         })
     }
 
     await writeNewResponse()
     await changeUpdateAtFieldForRequest()
```

Comment:
`changeUpdateAtField` is not necessary anymore, that functionality was moved to a cloud function

Reply:
This has to be fixed